File: app/src/main/java/com/intelegain/agora/model/KnowledgebaseProjectName.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by maninder on 21/11/17.
 */

public class KnowledgebaseProjectName {
    @SerializedName("ProjectData")
    @Expose
    public List<ProjectDatum> projectData = null;


    public class ProjectDatum {

        @SerializedName("projId")
        @Expose
        public Integer projId;
        @SerializedName("projName")
        @Expose
        public String projName;
        @SerializedName("custId")
        @Expose
        public Integer custId;
        @SerializedName("projDesc")
        @Expose
        public Object projDesc;

    }
}

File: app/src/main/java/com/intelegain/agora/model/KnowledgebaseProjectName.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by maninder on 21/11/17.
 */
class KnowledgebaseProjectName {
    @SerializedName("ProjectData")
    @Expose
    var projectData: List<ProjectDatum>? = null

    inner class ProjectDatum {
        @SerializedName("projId")
        @Expose
        var projId: Int? = null
        @SerializedName("projName")
        @Expose
        var projName: String? = null
        @SerializedName("custId")
        @Expose
        var custId: Int? = null
        @SerializedName("projDesc")
        @Expose
        var projDesc: Any? = null
    }
}